### PR TITLE
Add support for node v14.*

### PR DESCRIPTION
**Pull request by fabianrbz**
## Description

Bump node version to `< 15` and sets nvmrc to `14.15.4`  -Same as GH actions and heroku-

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -12,6 +12,7 @@
     "@babel/plugin-proposal-throw-expressions": "^7.12.1",
     "@babel/plugin-syntax-dynamic-import": "^7.0.0",
     "@babel/plugin-syntax-import-meta": "^7.10.4",
+    "@babel/polyfill": "^7.12.1",
```
Comment (mheap) [inline package.json:15]: Was this intentionally added?

Comment (fabianrbz) [reply package.json:15]: yes, just because we are including it in the pack. We can actually remove this requirement.

Comment (fabianrbz) [reply package.json]: Removed

**Review by bencgreenberg — APPROVED** [review]
Discussed with @fabianrbz on Slack, specifically our removal of the `webpacker` package, because we don't need it with the redundant inclusion of the `@rails/webpacker` package. This looks good, LGTM!

**Pull request merged**